### ISSUE TITLE
latest fixes not released and pushed to npm

### ISSUE THREAD

**Issue by DougReeder** [NikolaySuslov/aframe-croquet-component#13]
PR #11 (Commit 19ffa17062bdcc3fdab87976791a10911d661cf1) is not present in the code on npm

Please push out another release!

**Comment by NikolaySuslov** [comment]
Dear Doug! Sorry for had been missing that. Done.

**Comment by DougReeder** [comment]
Thanks!


> On Aug 3, 2024, at 6:23 PM, Nikolay Suslov ***@***.***> wrote:
> 
> 
> Dear Doug! Sorry for had been missing that. Done.
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID: ***@***.***>

Doug Reeder
***@***.***
Get routine info out of your head with Notes Together! Never spend time tidying up — unless you want to! https://notestogether.hominidsoftware.com/
My gender pronouns are: he, him, his.

**Comment by DougReeder** [comment]
I’m sorry to bother you, but I’m stumped about this.  In my current project, position propagates to other client, but scale does not.   Can you suggest a way to debug this?

https://beta.hominidsoftware.com/group-data-viewer/

These keys change the horizontal size: [ ]
These keys change the vertical size and adjust the position: { }
On Oculus Touch controllers, the A, B, X and Y buttons change the sizes.


Doug Reeder
***@***.***
Get routine info out of your head with Notes Together! Never spend time tidying up — unless you want to! https://notestogether.hominidsoftware.com/
My gender pronouns are: he, him, his.

**Comment by NikolaySuslov** [comment]
Hello Doug!

I think I have found the problem. Your code is fine. That is connected with
message sends limits, defined by the Reflector: "Sends to reflector are at
or above recommended limit of 20 within one second". So, setting attributes
are already throttled by aframe-croquet-component, but in your code there
is presented a sequential evaluation and that "line-by-line" of setting
'scale' and then just 'position' is not throttled.
(at line 88)
presentation.setAttribute('scale', scale);
presentation.setAttribute('position', offset);

So, the solution could be to use some sort of delayed evaluation using
setTimeout or libs with structured concurrency and effects for JavaScript.
I have checked this on default scene from the component with a simple cube
scene, and this works.

presentation.setAttribute('scale', scale);
setTimeout(() => {presentation.setAttribute('position', offset)}, 50);

Hope that helps.

Best regards,
Nikolay
